tests/TestCTimers.cpp

```diff
@@ -76,8 +76,6 @@ void TestCTimers::testCountdownCompletion()
     timers.addCountdown(countdownName.toStdString(), countdownDesc.toStdString(), countdownTimeMs);
     QString countdownsListBefore = QString::fromStdString(timers.getCountdowns());
     QVERIFY(countdownsListBefore.contains(countdownName)); // Verify the added countdown is present
-    QString countdownsList = QString::fromStdString(timers.getCountdowns());
-    QVERIFY(countdownsList.contains("(up for - 0:00, left - 0:10)"));
```

Comment:
This QVERIFY failed around than 10% of runs. I don't know why. Maybe sometimes it takes a millisecond or so to see the string in the list? So waiting a bit would fix it but I do not want to use QWAIT in tests because currently our tests take 4.4sec and using qwait a lot will make tests slow. I have tried mocking the clock but since this timer is based on real system clock it is not possible to mock, the actual implementation should also be changed to a mockable clock.

What do you suggest to do here?

Reply:
Are you saying it failed in 10% of your runs locally or 10% of GH runs?

Reply:
It sometimes failed on the server, on GH